Automattic/wp-calypso
# Reader: remove old post component

What I did:
```bash
git rm client/reader/stream/post.jsx
git mv client/reader/stream/refresh-post.jsx  client/reader/stream/post.jsx
```
- Sadly GH doesn't display this prettily.
- Also removed all traces of feature-flags from `stream.jsx`.

To Test
------
Browse through some of your streams and make sure posts look refreshed, x-posts as well.
